### PR TITLE
Fix: Remove the  `Format`, `Log` and `Controller` ignore errors.

### PR DESCRIPTION
**Pull request by kai890707**
**Description** 
See #7731
 
 **Checklist:**
 
 * [x]  Securely signed commits
 * [ ]  Component(s) with PHPDoc blocks, only if necessary or adds value
 * [ ]  Unit testing, with >80% coverage
 * [ ]  User guide updated
 * [x]  Conforms to style guide

### CODE REVIEW

File: system/Log/Exceptions/LogException.php [hunk 1]
```diff
@@ -15,11 +15,17 @@
...
+    /**
+     * @return LogException
+     */
     public static function forInvalidLogLevel(string $level)
     {
         return new static(lang('Log.invalidLogLevel', [$level]));
     }
 
+    /**
+     * @return LogException
```
Comment (MGatner) [inline system/Log/Exceptions/LogException.php:27]: `static` here and below.

**Review by MGatner — CHANGES_REQUESTED** [review]

File: system/Log/Exceptions/LogException.php (hunk 1 above)
Comment (kai890707) [reply system/Log/Exceptions/LogException.php]: done.

**Review by paulbalandan — APPROVED** [review]

**Review by kenjis — APPROVED** [review]
LGTM!

**Comment by MGatner** [comment]
🥳

**Pull request merged**